### PR TITLE
css_comments file added

### LINKED ISSUE #2923: [Create New Issue]: CSS Comments
### Description

In computer programming, a comment is a programmer-readable explanation or annotation in the source code of a computer program. They are added with the purpose of making the source code easier for humans to understand. I will be creating documentation regarding comments in CSS.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Sharayu1071**
<hr>

## Description 📜

I added documentation on CSS Comments.
Fixes #2923

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Sharayu1071** [comment]
Can you please review my PR!

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+- Comments are used to explain the code, and may help when you edit the source code at a later date.
+
+- A CSS comment is used to add explanatory notes to the code or to prevent the browser from interpreting specific parts of the style sheet. By design, comments have no effect on the layout of a document.
+
+- The /* */ comment syntax is used for both single and multiline comments. There is no other way to specify comments in external style sheets.
+
+- A CSS comment is placed inside the <style> element, and starts with /* and ends with */  
+
+<br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:14]: Why?

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Example-1:
+
+```jsx
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:18]: It's not JSX it's CSS.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+/* This is a single-line comment */
+
+p {
+  color: red;
+}
+
+```
+
+<br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:27]: Not needed.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Example-2:
+
+
+```jsx
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:32]: CSS

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+## *Reference:*
+   - Documentation:
+     - <a href="https://www.w3schools.com/css/default.asp">W3Schools</a> 
+     - <a href="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics">MDN Web Docs</a>
+<br>
+
+## Happy Coding :)
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:89]: Remove them.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+<br>
+
+
+
+## *Reference:*
+   - Documentation:
+     - <a href="https://www.w3schools.com/css/default.asp">W3Schools</a> 
+     - <a href="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics">MDN Web Docs</a>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:86]: Never use W3Schools.
Use Markdown not HTML.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
+
```
[the remark this answers is not in the document]
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:1]: Done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
+
+<h1 align="center"><b>CSS Comments</b></h1>
+<br>
+<br>
```
[the remark this answers is not in the document]
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+<br>
+
+- Comments are used to explain the code, and may help when you edit the source code at a later date.
+
+- A CSS comment is used to add explanatory notes to the code or to prevent the browser from interpreting specific parts of the style sheet. By design, comments have no effect on the layout of a document.
+
+- The /* */ comment syntax is used for both single and multiline comments. There is no other way to specify comments in external style sheets.
+
+- A CSS comment is placed inside the <style> element, and starts with /* and ends with */
```
[the remark this answers is not in the document]
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: Done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 1 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: removed

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 2 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 4 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 6 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 5 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: removed

Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md]: @praveenscience I've made all the changes that you've asked for. Could you please review the changes and merge it?

**Comment by Sharayu1071** [comment]
@praveenscience I've made all the changes that you've asked for. Could you please review the changes and merge it?

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+*/
+
+p {
+  color: pink;
+}
+```
+
+## combination HTML and CSS Comments for proper understanding ##
+
+```css
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:35]: Should be HTML and can you please indent the code?

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (praveenscience) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:1]: This seems like not done.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
+
+## CSS Comments
+Comments are used to explain the code, and may help when you edit the source code at a later date.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:3]: Add a line space before here.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
+
+## CSS Comments
+Comments are used to explain the code, and may help when you edit the source code at a later date.
+A CSS comment is used to add explanatory notes to the code or to prevent the browser from interpreting specific parts of the style sheet. By design, comments have no effect on the layout of a document.
+The /* */ comment syntax is used for both single and multiline comments. There is no other way to specify comments in external style sheets.
+A CSS comment is placed inside the <style> element, and starts with /* and ends with */
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:6]: Give space before these lines.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
+
+## CSS Comments
+Comments are used to explain the code, and may help when you edit the source code at a later date.
+A CSS comment is used to add explanatory notes to the code or to prevent the browser from interpreting specific parts of the style sheet. By design, comments have no effect on the layout of a document.
+The /* */ comment syntax is used for both single and multiline comments. There is no other way to specify comments in external style sheets.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:5]: Make it inside inline code: `/* */` like this.

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+<!-- HTML Comment -->
+<p>Hello World!</p>
+<p>This paragraph is styled with CSS.</p>
+<p>CSS comments are not shown in the output.</p>
+</body>
+</html>
+```
+  
+## Reference:
+- https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:56]: Not needed, but yea, let's keep it.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 11 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:3]: added

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 12 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:6]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 13 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:5]: done

File: Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md (hunk 14 above)
Comment (Sharayu1071) [reply Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md:56]: okay sir!

**Review by praveenscience — APPROVED** [review]
LGTM 👍

**Comment by Sharayu1071** [comment]
okay sir

**Pull request merged**